dev/service-registry/play-integration-javadsl/src/main/scala/com/lightbend/lagom/play/LagomPlayModule.scala

```diff
@@ -61,6 +61,9 @@ class LagomPlayModule extends Module {
     maybeServiceInfoBinding
   }
 
+  @deprecated(message = "prefer method using typesafe Config instead", since = "1.4.0")
+  private def prepareServiceInfoBinding(config: Configuration): Option[Binding[ServiceInfo]] = prepareServiceInfoBinding(config.underlying)
```

Comment:
It also isn't necessary to deprecate anything declared private, since by definition all uses are local and can be updated.